src/hw4/hw4.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+// 1. Задача на селектори:
+// Для сторінки з вебінару (лінк на гіт) напишіть селектори для наступних елементів:
+// - для елементу з текстом 'Навігація по DOM дереву'
+// - для першого елементу <section>
+// - для елементу списку з текстом 'Пункт 5'
+// - для елементу з класом 'hatredLevelBlock'
+// Кожен селектор має бути унікальним (тобто всі мають бути створені за допомогою різних методів) і має бути присвоєний якійсь змінній.
+// Приклад:
+//  let spanWithClass = document.querySelector('.hatredLevelCounter');
+
+let h2Class = document.getElementById('headerTwo');
+let firstSection = document.getElementsByClassName('firstSection');
+let text5 = document.querySelector('.text5');
+let divBlock = document.querySelector('.hatredLevelBlock');
```

Comment:
```suggestion
let text5 = document.querySelector('.text5');
let divBlock = document.querySelector('.hatredLevelBlock');
```
Однакові селектори, а мають всі бути різними

Reply:
Не виправлено